pyigm/clustering/survey_2d2pcf.py

```diff
@@ -219,22 +307,65 @@ def calc_xi_gg_transverse(self, s, nbins=None):
         self.DgRg_T = collapse_along_LOS(self.DgRg,nbins,s=s)
         self.RgRg_T = collapse_along_LOS(self.RgRg,nbins,s=s)
 
+
         # Calculate
-        self.xi_gg_T,self.xi_gg_T_err = W3(self.DgDg_T,self.RgRg_T,self.DgRg_T,self.DgRg_T,
-                                           Ndd=self.nDgDg,Nrr=self.nRgRg,Ndr=self.nDgRg,Nrd=self.nDgRg)
+        self.xi_gg_rperp,self.xi_gg_rperp_err = W3(self.DgDg_T,self.RgRg_T,self.DgRg_T,self.DgRg_T,
+                                           self.nDgDg,self.nRgRg,self.nDgRg,self.nDgRg)
+        # Finish
+        if nbins is None:
+            self.W_gg_T = self.xi_gg_rperp * 2 * self.tbinedges[-1]
```

Comment:
I do not understand this. Note that if `self.wrapped` is True in the field class, a factor of 2 is incorporated implicitly in all the pair counts along the line of sight.

Reply:
Well, it follows from Equation 21 of Tejos+14.

While we have wrapped the data to 'double' the number of pairs in a given rlos bin,
we have done the same for the Randoms.  So we are essentially evaluating from 0 to rlos_max.

Given the way that W_gg_T is defined, I think the factor of 2 is right.
But let's make sure we agree @ntejos

Reply:
I agree now with the factor of two, but shouldn't be `self.rbinedges[-1]` instead of `self.tbinedges[-1]`?